contrib/exporters/secadvisor/mod/transform_test.go

```diff
@@ -22,14 +22,70 @@ import (
 	"runtime"
 	"testing"
 	"time"
-
+	"io/ioutil"
 	cache "github.com/pmylund/go-cache"
-
+	"github.com/skydive-project/skydive/config"
 	"github.com/skydive-project/skydive/common"
-	"github.com/skydive-project/skydive/contrib/exporters/core"
+	//"github.com/skydive-project/skydive/contrib/exporters/core"
 	"github.com/skydive-project/skydive/flow"
 )
 
+const testConfig = `---
+host_id: ""
+analyzers:
+  - 127.0.0.1:8082
+pipeline:
+  analyzer:
+    subscriber_url: ws://127.0.0.1:8082/ws/subscriber/flow
+  classify:
+    cluster_net_masks:
+      - 10.0.0.0/8
+      - 20.0.0.0/16
+      - 192.168.0.0/16
+      - 127.0.0.1/16
+      - 172.17.0.0/16
+  filter:
+    excluded_tags:
+      - other
+      # - internal
+      # - egress
+      # - ingress
+  transform:
+    type: sa
+    sa:
+      # exclude_started_flows: true
+      extend:
+        - CA_Name=G.V().Has('RoutingTables.Src','{{.Network.A}}').Values('Host')
+        - CB_Name=G.V().Has('RoutingTables.Src','{{.Network.B}}').Values('Host')
+  encode:
+    type: json
+    json:
+      pretty: false
+  compress:
+    type: none
+  store:
+    type: stdout
+`
+
+func initConfig(conf string) error {
+	f, _ := ioutil.TempFile("", "extend_gremlin_test")
+	b := []byte(conf)
+	f.Write(b)
+	f.Close()
+	err := config.InitConfig("file", []string{f.Name()})
+	return err
+}
+
+func (e *extendGremlin) populateExtendGremlin(f *flow.Flow, transformer *securityAdvisorFlowTransformer) {
```

Comment:
make a helper func testResolveToHost() to be called for A name and B name

Reply:
Not sure what you mean here.

Reply:
notice that you are performing the same 3 operations per Network.A and Network.B - rather use a helper function